MatrixSDK/Utils/MXTaskQueue.swift

```diff
@@ -48,6 +48,7 @@ public actor MXTaskQueue {
             assertionFailure("Failing to get value of the correct type should not be possible")
             throw Error.valueUnavailable
         }
+        previousTask = nil
```

Comment:
Unrelated and untestable change that clears a `Task` after it has been completed. Without this change everythign works as normal, but we keep a supended Task in memory unnecessarily